### DISCUSSION TITLE
Algoritmo

### DISCUSSION THREAD

**Discussion by mgstraface** [Sebastian-pz/ducker-api#69] [Q&A] [answered]
*Necesitamos crear una función que detecte la temática de los cuacks a los que el usuario le da like para sugerirle más cuacks de esos temas. 
¿se les ocurre cómo hacerlo?

**Comment by Sebastian-pz** [comment, marked as answer]
Se puede hacer leyendo el string e identificando palabras claves dentro de la cadena
